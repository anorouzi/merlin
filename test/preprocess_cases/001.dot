graph g1 {

h1[kind=host,ip="10.0.0.1"];
h2[kind=host,ip="10.0.0.2"];
s1[kind=switch, ip="11.0.0.1", id=3];

s1 -- h1 [sport=0, dport=0, cost=1];
s1 -- h2 [sport=1, dport=0, cost=1];
h1 -- h1 [sport=1, dport=2, cost=1];
h2 -- h2 [sport=1, dport=2, cost=1];
}
