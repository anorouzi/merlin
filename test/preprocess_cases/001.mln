forall srcIP = 192.168.1.1 or srcIP = 192.168.1.2 ->  .* at min(50 Mbps);
