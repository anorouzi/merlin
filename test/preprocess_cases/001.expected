exists srcIP = 192.168.1.1 ->  .* at min(50 Mbps);
exists srcIP = 192.168.1.2 ->  .* at min(50 Mbps);
